src/openrct2/actions/SetCheatAction.hpp

```diff
@@ -347,7 +348,7 @@ DEFINE_GAME_ACTION(SetCheatAction, GAME_COMMAND_CHEAT, GameActionResult)
             case CheatType::SetStaffSpeed:
                 return { { 0, 255 }, { 0, 0 } };
             case CheatType::ForceWeather:
-                return { { 0, 8 }, { 0, 0 } };
+                return { { 0, WEATHER_COUNT }, { 0, 0 } };
```

Comment:
```suggestion
                return { { 0, WEATHER_COUNT - 1 }, { 0, 0 } };
```

There are 9 different values, with 0 being the lowest and 8 being the highest.

Reply:
Thanks :)

Reply:
Thanks. I do this, to begin with, must have undone at some point.